src/js/types/networks/cardano.js

```diff
@@ -157,6 +157,7 @@ export type CardanoSignTransaction = {
     networkId: number,
     auxiliaryData?: CardanoAuxiliaryData,
     scriptDataHash?: string,
+    collateral?: CardanoInput[],
```

Comment:
Same as in FW. I'd rather call all the collateral related variables `collateralInput`.